src/testrel.jl

```diff
@@ -188,7 +188,8 @@ call.
 """
 macro test_rel(args...)
     # Arguments need to be escaped individually, not all at the same time.
-    kwargs = [esc(a) for a in args]
+    # `setup` keyword is silently ignored, for compatibility with our old internal @test_rel.
+    kwargs = [esc(a) for a in args if !(a isa Expr) || (a.args[1] != :setup)]
```

Comment:
maybe clearer?
```suggestion
    kwargs = [esc(a) for a in args if !(a isa Expr && a.args[1] == :setup)]
```

Reply:
I was considering ignoring *all* unknown kwargs, for similar reasons. That would be less specific, but also not tie the implementations together.

Reply:
could do! i'm usually wary of things like that as it's not very user-friendly when it comes to typos

e.g. what happens if someone writes `@test_rel(query="...", expected_abort=true)` or `@test_rel(query="...", schema_input=Dict(...))`?

Reply:
I too am wary of that. I've also spent a lot of time recently debugging the problems that stem from unclear tests that at first glance seem to work. Let's go with not ignoring all unknown kwargs.

I think the suggested change above is clearer, since we're only checking one thing, `setup`.